--- ntrack-014/modules/Makefile.am~	2011-04-03 23:14:35.000000000 +0200
+++ ntrack-014/modules/Makefile.am	2011-04-04 20:03:42.835840583 +0200
@@ -54,21 +54,21 @@
 if HAVE_LIBNL1
 ntrack_libnl1_la_SOURCES = $(ntrack_libnl_sources)
 ntrack_libnl1_la_CFLAGS = $(LIBNL1_CFLAGS) $(ntrack_libnl_cflags)
-ntrack_libnl1_la_LIBADD = $(LIBNL1_LIBS)
+ntrack_libnl1_la_LIBADD = $(LIBNL1_LIBS) $(top_builddir)/common/libntrack.la
 ntrack_libnl1_la_LDFLAGS = $(ntrack_libnl_ldflags)
 endif
 
 if HAVE_LIBNL2
 ntrack_libnl2_la_SOURCES = $(ntrack_libnl_sources)
 ntrack_libnl2_la_CFLAGS = $(LIBNL2_CFLAGS) $(ntrack_libnl_cflags)
-ntrack_libnl2_la_LIBADD = $(LIBNL2_LIBS)
+ntrack_libnl2_la_LIBADD = $(LIBNL2_LIBS) $(top_builddir)/common/libntrack.la
 ntrack_libnl2_la_LDFLAGS = $(ntrack_libnl_ldflags)
 endif
 
 if HAVE_LIBNL3
 ntrack_libnl3_la_SOURCES = $(ntrack_libnl_sources)
 ntrack_libnl3_la_CFLAGS = $(LIBNL3_CFLAGS) $(ntrack_libnl_cflags)
-ntrack_libnl3_la_LIBADD = $(LIBNL3_LIBS)
+ntrack_libnl3_la_LIBADD = $(LIBNL3_LIBS) $(top_builddir)/common/libntrack.la
 ntrack_libnl3_la_LDFLAGS = $(ntrack_libnl_ldflags)
 endif
 
--- ntrack-014/configure.ac~	2011-04-03 23:18:31.000000000 +0200
+++ ntrack-014/configure.ac	2011-09-17 09:04:33.941929805 +0200
@@ -170,9 +170,7 @@
 
 AM_CONDITIONAL(HAVE_QT4, test $ac_have_qt4 = 1)
 
-PKG_CHECK_MODULES(LIBNL3, libnl-3.0, ac_have_libnl3=1, ac_have_libnl3=0)
-AS_IF([test x$ac_have_libnl3 = x1],
-	[LIBNL3_LIBS="$LIBNL3_LIBS -lnl-route"])
+PKG_CHECK_MODULES(LIBNL3, [libnl-3.0 libnl-route-3.0], ac_have_libnl3=1, ac_have_libnl3=0)
 AC_SUBST(LIBNL3_CFLAGS)
 AC_SUBST(LIBNL3_LIBS)
 AM_CONDITIONAL(HAVE_LIBNL3, test $ac_have_libnl3 = 1)
